src/components/index.ts

```diff
@@ -22,6 +22,7 @@ export { default as Tabs } from './Tabs';
 export { default as TextArea } from './TextArea';
 export { TextField, MuiTextField } from './TextField';
 export * from './Typography';
+export * from './Banner/Banner';
 
```

Comment:
We should also add the other components here

Reply:
ah good call